benches/benches/bevy_ecs/world_get.rs

```diff
@@ -20,6 +28,10 @@ struct Sparse(f32);
 
 const RANGE: std::ops::Range<u32> = 5..6;
```

Comment:
This being set to 5..6 is a little weird. Not introduced here, but it really feels like this should be something more like 1..10.

I'd like to get a larger curve, and test at smaller entity counts too.

Reply:
I'd prefer leaving these sorts of changes to a separate PR.

Reply:
1..10 would make the benches a lot longer to run